### PR TITLE
changed ball to dx dy

### PR DESCRIPTION
**Pull request by YCGreen**

### CODE REVIEW

File: src/main/java/brickbreaker/model/Ball.java [hunk 1]
```diff
@@ -4,54 +4,45 @@
...
+        dy = 1;
+        speed = 5;
     }
 
-    public void setAngle(double angle)
-    {
-        this.angle = angle;
+    public void reverseBallAngle() {
+        dx *= -1;
+        dy *= -1;
```
Comment (andrewoid) [inline src/main/java/brickbreaker/model/Ball.java]: dx = -dx

File: src/main/java/brickbreaker/model/Ball.java [hunk 2]
```diff
@@ -4,54 +4,45 @@
...
-                * -speed);
-        // multiply by -speed since y-axis is inverted
-        x = (int) (x + xDirection);
-        y = (int) (y + yDirection);
+    public void setAngle(double angle) {
+        dx = Math.cos(angle);
+        dy = Math.sin(angle);
     }
 
     public void reflectOffWall(int x, int y)
```
Comment (andrewoid) [inline src/main/java/brickbreaker/model/Ball.java:34]: y parameter isn't used

File: src/main/java/brickbreaker/neuralnetworks/Simulation.java [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+import basicneuralnetwork.NeuralNetwork;
+import brickbreaker.model.Game;
+
+import javax.swing.Timer;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.util.*;
+
+public class Simulation {
+    private Game game;
```
Comment (andrewoid) [inline src/main/java/brickbreaker/neuralnetworks/Simulation.java]: Merge Game and Simulation

Comment (rtaub5) [reply src/main/java/brickbreaker/neuralnetworks/Simulation.java]: We should combine these two classes into one?

Comment (andrewoid) [reply src/main/java/brickbreaker/neuralnetworks/Simulation.java]: I think so.

Comment (rtaub5) [reply src/main/java/brickbreaker/neuralnetworks/Simulation.java]: Ok, thank you.

**Pull request merged**